docs/content/test_runner/Metrics.md

```diff
@@ -89,9 +89,30 @@ Don't know exact reason, delta between initial test suite and reported tests
 
 ### `no-file`
 
-Test report data pulled from device and then pushed to report service \
+Test report data pulled from device and then pushed to report service. \
 These are cases when file was not found on device for some reason
 
 ### `parse-errors`
 
 If test file was pulled, but there was a parsing error
+
+## `adb`
+
+Lower level metrics from AdbDevice, to observe device connection issues. \
+It is count metrics, aggregated by time windows
+
+### `get-sdk-property`.[ `success` / `attempt-fail` / `failure` ]
```

Comment:
> ### `get-sdk-property`.[ `success` / `attempt-fail` / `failure` ] {#get-sdk-property}

To make ToC cleaner

![Screenshot from 2021-02-08 13-58-08](https://user-images.githubusercontent.com/1104540/107210810-b6e12d00-6a15-11eb-9b20-716cdc960fb8.png)